### ISSUE TITLE
Chrome extension: button stuck on loading on all songs

### ISSUE THREAD

**Issue by ophirf15** [brkastner/SDownload#135]
when the button is clicked to download a song the button switches to "loading"
it stays like this forever.

**Comment by brkastner** [comment]
This is because Chrome recently started enforcing using SSL which SDownload does not currently use. However, Chrome also started blocking extension installations from outside the web store so I have no way to provide any update that is usable without downloading the beta version of Google Chrome.

I'm pretty busy with work and other projects so I don't have the time to rewrite the extension in Firefox. I think I may have found someone to do it but if that falls through I'm happy to tell someone else who is capable how to go about finishing the Firefox version.

tl;dr: An update to Chrome will not be coming, hopefully a working Firefox version will come out in the future but the ETA is unknown.
